assets/refCtrl.js

```diff
@@ -87,9 +66,6 @@ module.exports = function ($scope, $filter, io) {
   $scope.modSaveProfile = function () {
     $scope.ok.modSaveProfile = false;
     $scope.spin.modSaveProfile = true;
-    if (!$scope.user.isMod) {
```

Comment:
The buttons don't appear if the user is not a mod. This check is clientside anyway, so it didn't have any security value.